src/test/java/SkeltalTest.java

```diff
@@ -0,0 +1,37 @@
+import java.io.ByteArrayOutputStream;
+import java.io.PrintStream;
+
+import org.junit.jupiter.api.BeforeEach;
```

Comment:
I think the imports should be in alphabetical order according to the style guide. Not sure if it's mentioned in the Java coding standard, but you will run into problems in checkstyle if you do implement it.